### DISCUSSION TITLE
Special settings for production?

### DISCUSSION THREAD

**Discussion by LeoEckert02** [pocketbase/pocketbase#383] [Q&A]
I am hosting my Frontend (svelte; I‘m using the JS client to access pocketbase) and my backend on a digitalOcean droplet. I can reach both my backend and my frontend through the chosen URLs but frontend and backend can’t seem to communicate. Are there any special settings I have to use when going into production?

**Comment by ganigeorgiev** [comment]
There are no such "special production settings". You should be able to access your PocketBase collections from the frontend, as long as your Collection API rules allow that.

Could you open your browser's "Inspect tool" and check if there is a JS error somewhere logged in the console?
A sample of the network request/response you are trying to make would be also helpful.

You could also check the logs in the PocketBase Admin UI for any error requests (you could filter only the errors with `status >= 400`).
